checker.py

```diff
@@ -38,7 +38,7 @@ def get_cc_output(cc: str, file: Path, flags: str, cc_timeout: int) -> tuple[int
         "-Wall",
         "-Wextra",
         "-Wpedantic",
-        "-O1",
+        "-O2",
```

Comment:
O3 is probably better, and the opposite is true for sanitizers, O0 is better

Reply:
Done